app/js/containers/Seller/4_SellerMarginContainer.jsx

```diff
@@ -26,6 +30,11 @@ class SellerMarginContainer extends Component {
     } else {
       this.setState({ready: true});
     }
+
+    this.props.fetchPrices({
+      from: [this._getSymbol()],
+      to: [this.props.seller.currency]
```

Comment:
Why are those arrays?

Reply:
fetchPrice allows asking for the price for more than a single token symbol.  In this case _getSymbol() returns the seller selected asset, and currency, the fiat currency symbol (EUR, USD...)